Userland/Applications/TextEditor/TextEditorWidget.cpp

```diff
@@ -705,7 +705,7 @@ void TextEditorWidget::update_statusbar()
         }
         if (in_word) word_count++;
 
-        builder.appendff("        Selected: {} characters ({} {})", m_editor->selected_text().length(), word_count, word_count != 1 ? "words" : "word");
+        builder.appendff("        Selected: {} {} ({} {})", m_editor->selected_text().length(), m_editor->selected_text().length() == 1 ? "character" : "characters", word_count, word_count != 1 ? "words" : "word");
```

Comment:
Please squash this fix into the previous commit as it has nothing to do with this one.